certification/internal/policy/operator/deployable_by_olm_test.go

```diff
@@ -97,16 +102,25 @@ var _ = Describe("DeployableByOLMCheck", func() {
 			OperatorSdkBVReport: report,
 		}
 
+		deployableByOLMCheck = *NewDeployableByOlmCheck(&fakeEngine)
+		scheme := apiruntime.NewScheme()
+		Expect(operatorv1.AddToScheme(scheme)).To(Succeed())
+		Expect(operatorv1alpha1.AddToScheme(scheme)).To(Succeed())
+		Expect(imagestreamv1.AddToScheme(scheme)).To(Succeed())
+		Expect(rbacv1.AddToScheme(scheme)).To(Succeed())
+		client = fake.NewClientBuilder().
+			WithScheme(scheme).
+			WithObjects(&csv, &csvDefault, &csvMarketplace, &ns, &secret, &sub).
+			WithLists(&pods, &isList).
+			Build()
+		deployableByOLMCheck.client = client
```

Comment:
Okay good, so this was the goal of being able to adjust the client in the check.

Reply:
Yep. Bye bye FakeOpenshiftEngine and BadOpenshiftEngine.